remote-access/access-over-Internet/README.md

```diff
@@ -2,10 +2,39 @@
 
 You can connect to your Raspberry Pi over the internet from another computer or a mobile device. There are a number of ways to do this, which we cover below.
 
-## Port forwarding
+## Port forwarding (IPv4)
 
 One method is to set up port forwarding on your router. To do this, you must change the configuration of your router to forward all inbound traffic from the internet on a specific port to the local IP address of your Raspberry Pi. Most routers have this feature available. However, every router is different so you will need to consult your router's user manual for instructions. The settings can be tricky if your Pi is behind a firewall or if there is more than one router. One disadvantage of port forwarding is that it exposes a network port on your private LAN to the public internet. This is a known security vulnerability and must be managed carefully.
 
+## Introduction for IPv6 newcomers
+
+There are big differences in handling IPv4 and IPv6 for remote access in residential environments(dsl,cable,fibre, maybe mobile too).
+With IPv4 it was and is still very common to use the public address of the router in combination with port forwarding. The router is the only device with a public address. So the router gets a packet and translates it (address and port) to a private IP address on the lan. Also the router is (mostly) responsible for getting a (nice) DNS name.
+With IPv6 some things have changed. Depending on your isp (different types of ds-lite and dualstack) you don't get a public IPv4 address anymore.
```

Comment:
What's ds-lite? When I google for it I only get results about Nintendo :confused:

Reply:
That' also a problem with the regional/language settings of google. I mean the technical term, defined in https://tools.ietf.org/html/rfc6333 .
It is not the only one, there are different implementations e.g. from greece
https://blog.apnic.net/2018/07/04/a-large-scale-lw4o6-deployment-for-greek-broadband-users/ 
They have on common, that IPv6 is usable in both directions, IPv4 only as client

Reply:
Perhaps it's worth mentioning the phrase "Dual-Stack Lite" at least once, so people have more context?